test/OpenTelemetry.Exporter.Console.Tests/ConsoleLogRecordExporterTest.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace OpenTelemetry.Exporter.Console.Tests;
+
+public class ConsoleLogRecordExporterTest
+{
+    [Fact]
+    public void VerifyExceptionAttributesAreWritten()
+    {
+        using var writer = new StringWriter();
+        System.Console.SetOut(writer);
+        using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+            .AddSource("*")
+            .ConfigureResource(resource =>
+                resource.AddService(
+                    serviceName: "Test_VerifyExceptionAttributesAreWritten",
+                    serviceVersion: "1.0.0"))
+            .AddConsoleExporter()
+            .Build();
+
+        using ILoggerFactory factory = LoggerFactory
+            .Create(builder => builder.AddOpenTelemetry(n =>
```

Comment:
nit: Use more intuitive variable names.
```suggestion
            .Create(builder => builder.AddOpenTelemetry(logging =>
```